rss/atomparser.cpp

```diff
@@ -119,8 +119,10 @@ Item AtomParser::parse_entry(xmlNode* entryNode)
 		} else if (node_is(node, "link", ns)) {
 			std::string rel = get_prop(node, "rel");
 			if (rel == "" || rel == "alternate") {
-				it.link = newsboat::utils::absolute_url(
-						base, get_prop(node, "href"));
+				if (it.link.empty() || it.link.substr(0, 4) != "http") {
```

Comment:
Please use `utils::is_http_url()` instead of `substr`, as it's more robust; for example, `substr` will allow `httpx://` while our utility function wouldn't.